doc/features/connection-pooling/README.md

```diff
@@ -1,23 +1,15 @@
 # Connection pooling
 
-The driver maintains one or more connections opened to each Cassandra node selected by the load-balancing policy.
+The driver maintains one or more connections opened to each Apache Cassandra node selected by the load-balancing policy.
 The amount of connections per host is defined in the pooling configuration.
 
 ## Default pooling configuration 
 
-The default number of connections per host depends on the version of the Cassandra cluster.
-
-Cassandra versions 1.2 and 2.0 allow the clients to send up to `128` requests without waiting for a response per
-connection. Higher versions of Cassandra ( that is 2.1 or greater) allow clients to send up to `32768` requests
-without waiting for a response.
-
-By default, the driver maintains two open connections to each host in the local datacenter and one to each host in a
-remote datacenter for Cassandra 1.2 or 2.0 and one connection to each host (local or remote) for Cassandra 2.1 or
-greater.
+The default number of connections per host depends on the version of the Apache Cassandra cluster. When using the driver to connect to modern server versions (Apache Cassandra 2.1 and above), the driver uses one connection per host.
 
 ## Setting the number of connections per host 
 
-You can set the number of connections per host in the pooling configuration:
+If needed, you can set the number of connections per host depending on the distance, relative to the driver instance, in the `pooling` configuration:
```

Comment:
Same suggestion as in https://github.com/datastax/nodejs-dse-driver/pull/55/files#r156555947

Reply:
Nevermind, all the documentation is using personal sentence. So, its best to keep same standard